pkg/server/login/templates.go

```diff
@@ -264,26 +269,26 @@ select.pf-c-form-control.pf-m-success { --pf-c-form-control--PaddingRight: var(-
               </div>
               <div class="pf-c-form__group">
                 <label class="pf-c-form__label" for="inputUsername">
-                  <span class="pf-c-form__label-text">Username</span>
+                  <span class="pf-c-form__label-text">{{ .Locale.Username }}</span>
                   <span class="pf-c-form__label-required" aria-hidden="true">*</span>
                 </label>
                 <input type="text" class="pf-c-form-control" id="inputUsername" placeholder="" tabindex="1" autofocus="autofocus" type="text" name="{{ .Names.Username }}" value="{{ .Values.Username }}">
               </div>
               <div class="pf-c-form__group">
                 <label class="pf-c-form__label" for="inputPassword">
-                  <span class="pf-c-form__label-text">Password</span>
+                  <span class="pf-c-form__label-text">{{ .Locale.Password }}</span>
                   <span class="pf-c-form__label-required" aria-hidden="true">*</span>
                 </label>
                 <input type="password" class="pf-c-form-control" id="inputPassword" placeholder="" tabindex="2" type="password" name="{{ .Names.Password }}" value="">
               </div>
               <div class="pf-c-form__group pf-m-action">
-                <button class="pf-c-button pf-m-primary pf-m-block" type="submit" tabindex="3">Log in</button>
+                <button class="pf-c-button pf-m-primary pf-m-block" type="submit" tabindex="3">{{ .Locale.LogIn }}</button>
               </div>
             </form>
           </div>
         </main>
         <footer class="pf-c-login__footer">
-          <p>Welcome to OKD.</p>
+          <p>{{ .Locale.WelcomeTo }} OKD</p>
```

Comment:
I think we lost the product name here.

Reply:
did we ? I still see the `OKD` there, we just pulled it out of the double curly brackets.

Reply:
Sorry, I missed that. Really you're not supposed to concat strings, instead making them a parameter in the message so the order of the words can change. That might be tricky here.

Reply:
@rebeccaalpert That doesn't work here since it's a Go template and not using i18next :/

Reply:
Bleh. Aiko did raise this as an issue re: ordering.

Reply:
> Sorry, I missed that. Really you're not supposed to concat strings, instead making them a parameter in the message so the order of the words can change. That might be tricky here.

@jhadvig how will we this need to be structured to handle translation changes to word order? eg. if  locale_jp becomes `OKD {{ .Locale.WelcomeTo }}`